tsconfig.json

```diff
@@ -7,7 +7,6 @@
         "target": "es5",
         "module": "commonjs",
         "moduleResolution": "node",
-        "importHelpers": true,
```

Comment:
By removing this line, we revert the value of `importHelpers` to default one - `false`

So what's the difference? 

Importing helpers results in using `ts-lib` for different functions, such as `exportingStar`. By disabling `importHelpers` we force TS to generate JS code, instead of requiring `ts-lib`:

`importHelpers: true`:
```
(0, tslib_1.__exportStar)(require("./admin/admin.types"), exports);
(0, tslib_1.__exportStar)(require("./article/article.types"), exports);
(0, tslib_1.__exportStar)(require("./common/common.types"), exports);
(0, tslib_1.__exportStar)(require("./company/company.types"), exports);
```

`importHelpers: false`:
```
var __exportStar = (this && this.__exportStar) || function(m, exports) {
    for (var p in m) if (p !== "default" && !Object.prototype.hasOwnProperty.call(exports, p)) __createBinding(exports, m, p);
};
```